source/DERIVATIVES.md

```diff
@@ -12,12 +12,12 @@ The outputs of fMRIprep can be found under the folder of each dataset (e.g. `mov
 - `ses-*/func` containing for each fMRI run of that session file prefixed with:
   - `_boldref.nii.gz` : a BOLD single volume reference.
   - `_*-brain_mask.nii.gz` : the brain mask in fMRI space.
-  - `_*-preproc_bold.nii.gz` : the preprocessed BOLD timeseries.
-  - `_*-confounds_regressors.tsv` : a tabular tsv file, containing a large set of confounds to use in analysis steps (eg. GLM). 
-  
+  - `_*-preproc_bold.nii.gz` : the preprocessed BOLD time series.
+  - `_*-confounds_regressors.tsv` : a tabular tsv file, containing a large set of confounds to use in analysis steps (e.g. GLM).
+
 ### Recommended preprocessing strategy
-The confounding regressors are correlated, thus it is recommended to use a subset of these regressors. Also note that preprocessed time series have not been corrected for any confounds, but simply realigned in space, and it is therefore critical to regress some of the available confounds prior to analysis. For python users, we recommend using [nilearn](https://nilearn.github.io) and the tool [load_confounds](https://github.com/SIMEXP/load_confounds) to load confounds from the fMRIprep outputs, using with the `Params24` strategy. As the NeuroMod data consistently exhibits low levels of motion, we recommend against removing time points with excessive motion (aka scrubbing). Because of the 2 mm spatial resolution of the fMRI scan, there is substantial impact of thermal noise, and some amount of spatial smoothing is advisable. Our preliminary analyses suggest thah `smoothing_fwhm=8` in nilearn nifti maskers to work well. 
-  
+The confounding regressors are correlated, thus it is critical to only use a subset of these regressors. Also note that preprocessed time series have not been corrected for any confounds, but simply realigned in space, and it is therefore also critical to regress some of the available confounds prior to analysis. See the [fMRIprep documentation](https://fmriprep.org/en/stable/outputs.html#confounds) for details on available confound regressors. For python users, we recommend using [nilearn](https://nilearn.github.io) and the tool [load_confounds_strategy](https://nilearn.github.io/stable/modules/generated/nilearn.interfaces.fmriprep.load_confounds_strategy.html) to load confounds from the fMRIprep outputs, using with a standardized strategy. As the NeuroMod data consistently exhibits low levels of motion, we recommend against removing time points with excessive motion (aka scrubbing), and the `minimal` strategy available in nilearn is a reasonable choice. Because of the 2 mm spatial resolution of the fMRI scan, there is substantial impact of thermal noise, and some amount of spatial smoothing is advisable. Our preliminary analyses suggest that `smoothing_fwhm=8` in nilearn nifti maskers to work well.
+
```

Comment:
I am not convinced about suggesting a specific smoothing level as it highly depends on the analysis requirements. Also 8mm is quite aggressive and only suits activation or functional connectivity analyses

Reply:
I have edited to suggest fwhm=5 and mention 8 only for central structures. We could drop recommendations altogether as well, but at the same time we have looked at these data a lot so it may be useful for some users to have a starting point.